tests/container

```diff
@@ -147,5 +147,10 @@ lxc exec u1 -- mkdir /root/sys
 lxc exec u1 -- mount -t sysfs sysfs /root/sys
 lxc delete -f u1
 
+
+echo "==> Try cleanly stopping a container"
```

Comment:
This is going to be unreliable as the guest OS wont have booted yet and wont respond